docker-compose/config/mimoto-issuers-config.json

```diff
@@ -1,129 +1,40 @@
 {
   "issuers": [
     {
-      "credential_issuer": "Mosip",
-      "protocol": "OpenId4VCI",
+      "issuer_id": "CAR",
+      "credential_issuer": "CAR",
```

Comment:
CAR is their country usecase specific word, we have to use a generic term. Please refer.

https://github.com/mosip/inji-config/pull/656